bin/oneoff/back_populate_user_proficiencies

```diff
@@ -175,10 +175,14 @@ def compute_basic_proficiency_at(proficiency, line)
   return nil
 end
 
-CSV.foreach('/tmp/user_proficiency.csv').with_index do |line, line_num|
-  puts "PROCESSING: line #{line_num}..." if line_num % 1000 == 0
+line_num = 0
+user_id = 0
```

Comment:
Note that these variables expose the `CSV.foreach` context to the `rescue` and `ensure` statements.